src/Umbraco.Web.Website/Models/ProfileModelBuilder.cs

```diff
@@ -64,10 +64,10 @@ public ProfileModelBuilder WithCustomProperties(bool lookupProperties)
             Comments = member.Comments,
             IsApproved = member.IsApproved,
             IsLockedOut = member.IsLockedOut,
-            LastLockoutDate = member.LastLockoutDateUtc?.ToLocalTime(),
-            CreatedDate = member.CreatedDateUtc.ToLocalTime(),
-            LastLoginDate = member.LastLoginDateUtc?.ToLocalTime(),
-            LastPasswordChangedDate = member.LastPasswordChangeDateUtc?.ToLocalTime(),
+            LastLockoutDate = member.LastLockoutDate,
+            CreatedDate = member.CreatedDate.Date,
```

Comment:
I'm assuming the `.Date` wasn't intentional?
```suggestion
            CreatedDate = member.CreatedDate,
```

Reply:
No, good spot.